activities/views/posts.py

```diff
@@ -53,6 +54,22 @@ def get_context_data(self):
             "link_original": True,
             "ancestors": ancestors,
             "descendants": descendants,
+            "opengraph": {
+                "title": f"{self.post_obj.author.name} (@{self.post_obj.author.handle})",
+                "type": "article",
+                "published_time": self.post_obj.published.isoformat()
+                or self.post_obj.created.isoformat(),
+                "modified_time": self.post_obj.edited.isoformat()
+                or self.post_obj.published.isoformat()
+                or self.post_obj.created.isoformat(),
+                "description": self.post_obj.summary
+                or bleach.clean(self.post_obj.safe_content_local(), strip=True),
```

Comment:
There's a `core.html.strip_html` function I'd rather you use here.

Reply:
I looked at that, but it linkifies, which we can't accept here. OpenGraph requires bare text or it rejects AFAICT.

Should I put a boolean param on that function?

Reply:
Hmm, you could, or you could consider `html_to_plaintext` in that same module which will also turn <p> and <br> back into newlines if you wanted that.

Reply:
Ah, missed that. But since we switched to `|striptags` for now this may be mooted